cli/lib/kontena/cli/nodes/labels/remove_command.rb

```diff
@@ -12,7 +12,7 @@ class RemoveCommand < Kontena::Command
     def execute
       node = client.get("nodes/#{current_grid}/#{node_id}")
       data = { labels: Array(node['labels']).reject {|label| label_list.include?(label) } }
-      client.put("nodes/#{current_grid}/#{node['id']}", data)
+      client.put("nodes/#{current_grid}/#{node['name']}", data)
```

Comment:
Shouldn't `node['id']` work here?

unrelated-side-note:
Why are we using `PUT` to remove a node? :)